enumfields/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.admin.options import IncorrectLookupParameters
+from django.contrib.admin import FieldListFilter
+
+
+class EnumFieldListFilter(FieldListFilter):
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        super().__init__(field, request, params, model, model_admin, field_path)
```

Comment:
This super call needs to have `EnumFieldListFilter, self` to retain compatibility with Python 2.